bigbluebutton-client/resources/prod/lib/verto_extension.js

```diff
@@ -11,30 +11,24 @@ Verto = function (
   this.share_call = null;
   this.vertoHandle;
 
-  this.vid_width = 1920;
-  this.vid_height = 1080;
+  this.vid_width = window.screen.width;
+  this.vid_height = window.screen.height;
 
-  this.local_vid_width = 320;
-  this.local_vid_height = 180;
-  this.outgoingBandwidth;
-  this.incomingBandwidth;
-  this.sessid = null;
+  this.outgoingBandwidth = "default";
+  this.incomingBandwidth = "default";
+  this.sessid = $.verto.genUUID();
 
   this.renderTag = 'remote-media';
 
   this.destination_number = voiceBridge;
   this.caller_id_name = conferenceUsername;
-  this.caller_id_number = conferenceIdNumber;
+  this.caller_id_number = conferenceUsername;
 
   this.vertoPort = "8082";
   this.hostName = window.location.hostname;
   this.socketUrl = 'wss://' + this.hostName + ':' + this.vertoPort;
```

Comment:
Do the same as for webrtc voice where the ws goes thru nginx. This way, we don't need to configure FS with certificates.

this.socketUrl =  ('https:' == document.location.protocol ? 'wss://' : 'ws://')  + server + '/verto',

Create nginx config /etc/bigbluebutton/nginx/verto.nginx

location /verto {
        proxy_pass https://192.168.23.4:8082;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "Upgrade";
        proxy_read_timeout 6h;
        proxy_send_timeout 6h;
        client_body_timeout 6h;
        send_timeout 6h;
}